internal/api/integrations/k8s/k8s.go

```diff
@@ -76,16 +85,44 @@ func (k *K8sClient) ListAll(namespace string) ([]models.Release, error) {
 				Sumologic: annotations[annotationFor("sumologic")],
 			},
 			Code: models.SourceCode{
-				Github: annotations[annotationFor("code")],
+				Github: findVCSRepo(codeURL),
+				Ref:    image.Tag,
 			},
 			Artifacts: models.Artifacts{
 				Chart: models.HelmArtifact{
 					Path:    r.Spec.Chart.Path,
 					Version: r.Spec.Chart.Revision,
 				},
+				Docker: models.DockerArtifact{
+					Image: image.Repository,
+					Tag:   image.Tag,
+				},
 			},
 		})
 	}
 
 	return releases, nil
 }
+
+func findVCSRepo(url string) string {
+	arr := strings.Split(url, "/")
+	if len(arr) > 4 {
+		return arr[4]
+	}
+	return ""
+}
+
+func GetImageForRepo(repo string, vals map[string]interface{}, logger log.Logger) internal.Image {
```

Comment:
Instead of taking a logger argument, this func should return an `error` and let the caller log it. I'd also suggest returning `(*Image, error)` so we can return `nil` in the error case.